packages/shared/src/hooks/usePostModalNavigation.ts

```diff
@@ -64,14 +65,14 @@ export const usePostModalNavigation = (
         }
 
         setIsFetchingNextPage(false);
-        trackEvent({
-          origin: 'article modal',
-          event_name: 'navigate next',
-          target_id: item.post.id,
-        });
+        trackEvent(
+          postAnalyticsEvent('navigate next', item.post, {
```

Comment:
We should use here the current post, not the new one.
This makes more sense from the perspective of data analysis.